tests/lcs/core/engine/widget.livecodescript

```diff
@@ -80,3 +81,21 @@ end TestWidgetScriptObjectAccess
 on errorDialog pError
    put pError into gLastSentWidgetError
 end errorDialog
+
+//////////
+
+on TestWidgetThrowInOnCreate
+   set the lockErrorDialogs to true
+   create stack "WidgetThrowOnSaveTest"
+   set the defaultStack to "WidgetThrowOnSaveTest"
+
+   put empty into gLastSentWidgetError
+   
+   create widget "Test" as "com.livecode.lcs_tests.core.widgetthrow"
```

Comment:
@livecodeali I'm wondering if `errorDialog` should be centrally handled with a way to tell the test lib to collect a particular error rather than quit. Otherwise maybe pass if `pError` is not `error`?